integration-tests/src/test/java/tachyon/master/JournalIntegrationTest.java

```diff
@@ -616,4 +620,71 @@ private void rawTableTestUtil(FileInfo fileInfo) throws Exception {
 
     fsMaster.stop();
   }
+
+  @Test
+  @LocalTachyonClusterResource.Config(tachyonConfParams = {
+      Constants.SECURITY_AUTHENTICATION_TYPE, "SIMPLE",
+      Constants.SECURITY_AUTHORIZATION_PERMISSION_ENABLED, "true",
+      Constants.SECURITY_GROUP_MAPPING, FakeUserGroupsMapping.FULL_CLASS_NAME})
+  public void setAclTest() throws Exception {
+    TachyonURI filePath = new TachyonURI("/file");
+
+    System.setProperty(Constants.SECURITY_LOGIN_USERNAME, "tachyon");
+    OutStreamOptions op =
+        new OutStreamOptions.Builder(mMasterTachyonConf).setBlockSizeBytes(64).build();
+    mTfs.getOutStream(filePath, op).close();
+
+    mTfs.setAcl(filePath,
+        new SetAclOptions.Builder().setOwner("user1").setRecursive(false).build());
+    mTfs.setAcl(filePath,
+        new SetAclOptions.Builder().setGroup("group1").setRecursive(false).build());
+    mTfs.setAcl(filePath,
+        new SetAclOptions.Builder().setPermission((short) 0400).setRecursive(false).build());
+
+    FileInfo fileInfo = mTfs.getInfo(mTfs.open(filePath));
+
+    mLocalTachyonCluster.stopTFS();
+
+    aclTestUtil(fileInfo);
+    deleteFsMasterJournalLogs();
+    aclTestUtil(fileInfo);
+  }
+
+  private void aclTestUtil(FileInfo fileInfo) throws Exception {
```

Comment:
It is named to be consistent with other tests, such as `pinTestUtil`, `renameTestUtil`, etc. I can change its name in next updates if we still want.